packages/core/test/integration/json.test.ts

```diff
@@ -165,39 +177,75 @@ describe('JSON Querying', () => {
 
   if (dialect.supports.jsonOperations) {
```

Comment:
This support table should be split into:

- jsonExtraction.unquoted
- jsonExtraction.quoted

This way instead of expecting mssql to fail all of these tests, you can simply disable the tests that use the regular extract without unquote based on `support.jsonExtraction.quoted`

Reply:
I've added this in now.